### ISSUE TITLE
Mobile size for Wireframes is too big

### ISSUE THREAD

**Issue by hokienick** [topcoderinc/cab#38]
Wirefame designing on mobile should require small size like 320x480 not big ones like 1080x1920  as this track is about concepts not accurate visuals and it makes it more difficult for wireframers to implement ideas. Also more elements fit in a bigger page, than smaller, which might not look good on an actual design.

**Comment by hokienick** [comment]
There have been communication to Design Copilots about this.

**Comment by f0rc0d3r** [comment]
Last contest: https://www.topcoder.com/challenge-details/30057634/?type=design
states for Mobile size: 750px x 1334px
which looks too big in Axure.

New members use to ask about this size because this renders too big.

I think this should be re-opened and specify which is the solution. 
In this case mobile could has been stated for Axure like: 375x667 IMHO

**Comment by Ravijune** [comment]
Yes this isn't supposed to happen again as the issue was solved already. Would be great if this is mentioned again to the copilots before writing the WF spec.
Hey @f0rc0d3r, hope you informed this on the forum of the challenge so the copilot could update the spec with the new device dimensions.

**Comment by f0rc0d3r** [comment]
Sorry but I was late noticing that... I use to translate the size in my head directly... and now looking for something else I just went to the last WF which I new there was a mobile to design and I found this happening...
